packages/block-library/src/details/style.scss

```diff
@@ -0,0 +1,13 @@
+.wp-block-details {
+	::marker {
+		font-size: var(--wp--preset--font-size--medium, $default-font-size);
+	}
+	.wp-block-details__summary {
+		font-size: var(--wp--preset--font-size--medium, $default-font-size);
+		> * {
+			display: inline;
+			margin-left: 8px;
+			font-size: var(--wp--preset--font-size--medium, $default-font-size);
```

Comment:
Could we do away with these font-size declarations so that the typography block supports affects all text within a details block? Or at least have it so that H1, H2 etc sizes take effect?

Also, what do you think about having a `cursor:pointer` on the summary tag?